torch/_inductor/graph.py

```diff
@@ -491,6 +497,9 @@ def compile_to_module(self):
         for name, value in self.constants.items():
             setattr(mod, name, value)
 
+        for name, value in self.opaque_callable.items():
+            setattr(mod, name, value)
```

Comment:
Is this the right way to attach external callable to the compiled module?

Another alternative is to pass the external callable via "globals" or "locals" parameters when calling exec().

Reply:
Do we expected the generated wrapper script to be standalone runnable? 
Given the current implementation, it's no longer self-contained.

Reply:
Yeah having generated wrapper script be standalone runnable is a pretty cool feature for debugging, would be unfortunate to lose it.

Reply:
The more general concept here, is that we probably want more of a protocol for opaque callables than "it is any python callable, no holds barred". This protocol would encapsulate things like "can dump to be standalone runnable", for example.